.travis.yml

```diff
@@ -12,5 +12,5 @@ cache:
 install: true
 
 script:
-  - ./mvn-build-all.sh clean install jacoco:report
-  - mvn -U coveralls:report
+  - mvn -U clean install jacoco:report
```

Comment:
@btyoshikawaa 
No, I don't need it. I modified these files about jacoco. 01449544e5acc1855ce3f554c1dd5640e6464d84